apps/ledger-live-desktop/src/renderer/families/internet_computer/ManageNeuronFlowModal/steps/SelectFollowees.tsx

```diff
@@ -0,0 +1,197 @@
+        if (value.includes(followTopic)) {
+          return key;
+        } else {
+          return undefined;
+        }
+      })
+      .filter(Boolean) as string[]),
+  ]);
+
+  const onChangeFollowNeuronId = useCallback(
+    (value: string) => {
+      setFollowNeuronId(value);
+      if (value && /[^0-9]/.test(value)) {
+        setError(new Error("Invalid neuron ID, please enter a valid neuron ID"));
+      } else if (followees.includes(value)) {
+        setError(new Error("Neuron ID already in followees"));
+      } else {
+        setError(null);
+      }
+    },
+    [setFollowNeuronId, followees],
+  );
+
+  const onClickFollowNeuron = useCallback(() => {
+    const bridge = getAccountBridge(account, undefined);
+    const initTx = bridge.createTransaction(account);
+    onChangeTransaction(
+      bridge.updateTransaction(initTx, {
+        neuronId: neuron.id[0]?.id.toString(),
+        type: "follow",
+        followeesIds: followees,
+        followTopic,
+      }),
+    );
+    setLastManageAction("follow");
+    transitionTo("manageAction");
+  }, [
+    transitionTo,
+    account,
+    neuron.id,
+    followees,
+    followTopic,
+    onChangeTransaction,
+    setLastManageAction,
+  ]);
+
+  const onClickAddFollowee = useCallback(
+    (neuronId?: string) => {
+      const newFollowees = [...followees];
+      if (neuronId?.length) {
+        newFollowees.push(neuronId);
+      } else if (followNeuronId.length) {
+        newFollowees.push(followNeuronId);
+      }
+      setFollowees(newFollowees);
+      setFollowNeuronId("");
+    },
+    [followNeuronId, followees],
+  );
+
+  const onClickRemoveFollowee = useCallback((followee: string) => {
+    setFollowees(prev => prev.filter(f => f !== followee));
+  }, []);
+
+  return (
+    <Box>
+      <Box mb={4}>
+        <Text ff="Inter|SemiBold" fontSize={16}>
+          {t(`internetComputer.manageNeuron.followTopic.${followTopic}.title`)}
+        </Text>
+      </Box>
+      <Box style={{ gap: 5 }}>
+        <Label>{t("internetComputer.manageNeuronFlow.selectFollowees.title")}</Label>
+        <Box style={{ gap: 5, justifyContent: "flex-end" }}>
+          <Input
+            value={followNeuronId}
+            onChange={onChangeFollowNeuronId}
+            error={error}
+            placeholder={t("internetComputer.manageNeuronFlow.selectFollowees.placeholder")}
+          />
+          <Button
+            onClick={() => onClickAddFollowee()}
+            disabled={!!error}
+            style={{ marginLeft: "auto" }}
+            primary
+          >
+            {t("common.add")}
+          </Button>
+        </Box>
+      </Box>
+      <Divider my={4} width={"100%"} />
+      <Box style={{ gap: 10 }}>
+        <Text ff="Inter|SemiBold" fontSize={14}>
+          {t("internetComputer.manageNeuronFlow.selectFollowees.options")}
+        </Text>
+        <Box style={{ gap: 15 }}>
+          {Object.entries(KNOWN_NEURON_IDS).map(([key, value]) => (
```

Comment:
KNOWN_NEURON_IDS is a constant, so maybe create the array once outside the component.

```
const neuronIdsList = Object.entries(KNOWN_NEURON_IDS);
```